workout-logger/scripts/test_gemini_api.py

```
@@ -0,0 +1,308 @@
+import os
+import re
+import sys
+import time
+import urllib.error
+import urllib.request
+
+
+def load_env_file(filepath: str) -> None:
+    if not os.path.exists(filepath):
+        return
+    with open(filepath, "r", encoding="utf-8") as f:
```

Ruff (0.16.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)